src/Microsoft.DotNet.Tools.New/Program.cs

```diff
@@ -24,13 +30,27 @@ private static string GetFileNameFromResourceName(string s)
             return parts[parts.Length - 2] + "." + parts[parts.Length - 1];
         }
 
-        public int CreateEmptyProject()
+        public int CreateEmptyProject(Language language)
```

Comment:
After more thought, we have many places where we expect a full path as input, so I wouldn't worry about that.